packages/builder/package.json

```diff
@@ -62,6 +62,7 @@
     "@budibase/frontend-core": "0.0.1",
     "@budibase/shared-core": "0.0.1",
     "@budibase/string-templates": "0.0.1",
+    "@budibase/types": "0.0.1",
```

Comment:
I think this is a great thing for us to start doing. A lot of our constants are duplicated - there's no real reason not to bring in the types package. Especially if we're ever gonna implement TS in the parts of the builder that integrate with APIs etc. 

I believe we already pull it in transiently anyway via the `shared-core` dependency. Better to call it out explicitly like you have here.